### ISSUE TITLE
Layer opacity changes cell opacity instead

### ISSUE THREAD

**Issue by ByteSeb** [Orama-Interactive/Pixelorama#874]
**Pixelorama version:**
0.10.3-stable

**OS/device including version:**
Fedora Linux 38

**Issue description:**
Not sure if this is expected behavior, but I can't seem to change the opacity of a layer or selected frames. It just sets the visibility of a single selected cell instead.

**Steps to reproduce:**
Select a layer/group of cells. Move the opacity slider. Only one cell will be affected

[LayerOpacity.webm](https://github.com/Orama-Interactive/Pixelorama/assets/85042318/95ca74da-1d16-49b8-9ca7-5332c9733bb8)

**Comment by OverloadedOrama** [comment]
Yeah, this is intentional behavior. Layer opacity does not exist yet, only cel opacity. It's mostly a UI/UX issue, we need to figure out where exactly cel opacity should be, and where layer opacity should be, when we add it. Closing as this is being tracked on #587.
